src/test/java/com/easypost/BetaRateTest.java

```diff
@@ -17,6 +25,15 @@
 public class BetaRateTest {
     private static TestUtils.VCR vcr;
 
+    private static final MockStatelessRateResponse MOCK_STATELESS_RATE_RESPONSE = new MockStatelessRateResponse(
+            ImmutableList.of(
+                    new MockRate("6.07", "USPS", "First"),
```

Comment:
praise: this is better, thank you

thought: we may want to get ahead of the service level issue now since this is mocked? We could fix the `First` and `ParcelSelect` now while we are in here